db/migrate/20200618175923_delete_tracking_state_for_staged_users.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class DeleteTrackingStateForStagedUsers < ActiveRecord::Migration[6.0]
+  def change
```

Comment:
I think this should be split into `up`/`down` methods with the `down` method throwing an "NotReversibleMigration` error.